addons/event/views/event_views.xml

```diff
@@ -587,21 +587,32 @@
                     <field name="event_ticket_id"/>
                     <templates>
                         <t t-name="kanban-box">
-                            <div class="oe_kanban_card oe_kanban_global_click">
+                            <div class="oe_kanban_card oe_kanban_global_click" style="width:400px">
                                 <div class="float-right">
                                     <a class="btn btn-sm btn-success" string="Confirm Registration" name="action_confirm" type="object" states="draft" role="button">
-                                        <i class="fa fa-check" role="img" aria-label="Confirm button" title="Confirm Registration"/>
+                                        <i class="fa fa-check fa-7x" role="img" aria-label="Confirm button" title="Confirm Registration"/>
                                     </a>
-                                    <a class="btn btn-sm btn-success" string="Attended the Event" name="action_set_done" type="object" states="open" role="button">
-                                        <i class="fa fa-level-down" role="img" aria-label="Attended button" title="Attended the Event"/>
+                                    <a class="btn btn-sm btn-success" string="Confirm Attendance" name="action_set_done" type="object" states="open" role="button">
+                                        <i class="fa fa-user-plus fa-7x" role="img" aria-label="Attended button" title="Confirm Attendance"/>
                                     </a>
+                                    <div string="Attended" type="object" states="done" role="button">
+                                        <i class="fa fa-users fa-7x" role="img" aria-label="Attended" title="Attended"/>
+                                    </div>
+                                    <div string="Cancelled" type="object" states="cancel" role="button">
+                                        <i class="fa fa-user-times fa-7x" role="img" aria-label="Cancelled" title="Cancelled"/>
+                                    </div>
                                 </div>
                                 <div class="oe_kanban_content">
                                     <div class="o_kanban_record_top">
-                                        <b class="o_kanban_record_title"><field name="display_name"/></b>
+                                        <strong class="o_kanban_record_title"><span><t t-esc="record.name.value"/></span></strong>
                                     </div>
                                     <div class="o_kanban_record_body">
-                                        <field name="event_id"/>
+                                        <div class="row">
+                                            <field name="event_id" invisible="context.get('active_id')" />
```

Comment:
Why do you use the invisible attribute? Not sure I understand.

Reply:
As I understood, when we are in the single view mode, we must not show the "event name", misunderstanding?

Reply:
Yes I see it in the specifications now.

Changing the kanban card display based on the applied filter is not something we usually do and I fear it may give unexpected results / dirty code.

You can ignore this point in the specs, I will ping LNA about it.

Reply:
Alright, I put it back the default behavior.

Reply:
After discussing with TDE, yo can put this back but with a `invisible="context.get('default_event_id')"` instead.

We will allow this small customization as it helps the display for attendees.

Sorry for the confusion.

Reply:
No problem. Thanks for the review.